addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -378,7 +378,7 @@ $o-wblog-title-sizes-variants: (
 //------------------------------------------------------------------------------
 // Latest posts snippets
 //------------------------------------------------------------------------------
-.s_latest_posts, .s_latest_posts_big_picture {
```

Comment:
I don't think this file is compatible with old snippet ? Should not it be duplicated using our new system? There can't be only one commit for this PR when it is about moving stuff from theme + implementing new options. At least:

1. Add theme scss here (1 commit)
2. Move to 000.scss (1 commit)
3. New scss -> new version (1 commit)

Reply:
That is correct. In that case I might as well rename a bunch of those classes in the new version, as they have weird names (all start with s_ even though they're not top-level snippet classes), are all very long and redundant, and now the snippet can't even really be called "s_latest_posts" anymore since you can also choose "most viewed posts" in the options.

Reply:
Actually there is no real convention for classes used on snippet parts but I won't mind them starting by s_xxx_yyy_ where s_xxx_yyy is the name of the snippet. That's what we did with the product catalog for example: https://github.com/odoo/odoo/commit/03c52a23c1e5a182cce087ad6ad32a305d3e18aa#diff-38040433d5cfbefe351dd369b970f40bR5 (the snippet is named `s_product_catalog` and the parts begin by `s_product_catalog`). I like it that way but it is fine if it starts with `o_`. The important part is that the classes have a prefix, a clear name and the top level one *must* start with `s_`.

But yes, in this case, it could be nice to just take advantage of the new non-compatible options to deprecate old classes names.

Reply:
It's just weird to use such long and specific names and also use scss rule nesting. Currently, we have `.s_latest_posts_post_cover` nested in `.s_latest_posts_post` nested in `.s_latest_posts_big_picture` which generates this css: `.s_latest_posts_big_picture.s_latest_posts_post.s_latest_posts_post_cover` instead of a much more readable and just as explicit `.s_latest_posts.o_layout_big_picture.o_post.o_cover`, or simply using the last class as is `s_latest_posts_post_cover`, although at that point you lose rule priority and code locality, so in general I feel like favoring long and explicit top-level classes and using shorter (still explicit) non-redundant ones inside is the right choice. Thoughts?

Reply:
For naming, I would obviously agree... for a small project between two friends ;) Believe me, if every app used what you suggest, it will become a nightmare when it comes to grepping class names (actually, I already had to solve conflicts between class names defined in web, website and internal... for totally different use case and which happen totally by (mal)chance). We would have lots of o_small, o_big, o_left_image, o_image_left, which will all mean different things. Your snippet will have a far better chance at being broken... just because a stupid app made a generic rule targeting all 'o_cover' for example. Even inside our 50+ snippets, it will be a lot more easier to grep one class and find it directly, instead of finding it in 10 different snippets with 10 different meanings.

For nesting, it is in general better *not* to nest (smaller and more efficient css, easier to override, more declarative, etc). BS4 almost never nest for example. This however does not work with our compatibility system with `[data-vcss]`, so we are stuck with nesting.

Note: ideally snippets should not have many parts to target and/or style, probably rarely more than one level anyway.